### PR TITLE
Chore/changelogs

### PR DESCRIPTION
**Pull request by literat**
Prepare and fill package changelog files with missing content. From this point the changelogs will be filled automatically by lerna using lmc`s github preset.

### CODE REVIEW

File: packages/browserslist-config/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines and [LMC GitHub preset](https://github.com/lmc-eu/code-quality-tools/tree/main/packages/conventional-changelog-lmc-github) for a list of supported types.
+
+<a name="1.0.1"></a>
+
+## [1.0.1](https://github.com/lmc-eu/code-quality-tools/compare/@lmc-eu/browserslist-config@1.0.0...@lmc-eu/browserslist-config@1.0.1) (2021-12-18)
```
Comment (adamkudrna) [inline packages/browserslist-config/CHANGELOG.md:8]: Cool, link to diff is useful! Maybe I'd also expect a link to GitHub Releases page here. But I'm not sure it's applicable for monorepos.

File: packages/browserslist-config/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Introduce lincense and add credit to strv for inspiration ([a2b5a9e](https://github.com/lmc-eu/code-quality-tools/commit/a2b5a9e))
+
+<a name="1.0.0"></a>
+
+# 1.0.0 (2021-07-28)
+
+### Features
+
+- **browserslist-config:** Introduce Browserslist config ([a7b5e1e](https://github.com/lmc-eu/code-quality-tools/commit/a7b5e1e))
```
Comment (adamkudrna) [inline packages/browserslist-config/CHANGELOG.md:20]: Aha! This is where the package name is used 🙂.

**Review by adamkudrna — APPROVED** [review]
Just a few nits, but it looks good!

File: packages/browserslist-config/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines and [LMC GitHub preset](https://github.com/lmc-eu/code-quality-tools/tree/main/packages/conventional-changelog-lmc-github) for a list of supported types.
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/browserslist-config/CHANGELOG.md]: Ok, thanks :-)

**Pull request merged**